### PR TITLE
New: lint JSDocs

### PR DESCRIPTION
**Pull request by niedzielski**
When specified, JSDocs should be consistent:

• check-param-names: parameters documented must match those declared
• check-tag-names: forbid invalid JSDoc block tags
• check-types: forbid invalid types
• newline-after-description: forbid excess newlines after description
• require-param-type: parameters documented must specify type
• require-returns-type: return values documented must specify type

All rules new to this patch are emitted as warnings

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -357,6 +365,34 @@ module.exports = {
         }
     },
     "ecmaFeatures": {},
-    "extends": "eslint:recommended"
+    "extends": "eslint:recommended",
+    "settings": {
+        // https://www.npmjs.com/package/eslint-plugin-jsdoc#eslint-plugin-jsdoc-settings-alias-preference
+        "jsdoc": {
+            // one synonym for each ambiguity listed: http://usejsdoc.org/#block-tags
+            "tagNamePreference": {
```
Comment (niedzielski) [inline .eslintrc.js]: Feel free to amend these to what the team prefers. Hopefully we can be consistent one way or another! :]

**Comment by niedzielski** [comment]
> All rules new to this patch are emitted as warnings
Bump! Just wanted to highlight that for clients that do not limit warnings (the default behavior), this is a nonbreaking change. I think we should strive for consistency in our docs so I hope this can be merged :+1:

**Comment by Pchelolo** [comment]
@niedzielski Could you rebase? After I'll run it on our major repos to see the impact and if the damage is reasonable - merge!

**Comment by niedzielski** [comment]
@Pchelolo, thanks! I've merged master to preserve comment history but let me know if you prefer I squash it.

**Comment by Pchelolo** [comment]
Ok, I've checked several services and the effect of this is moderate. One last thing @niedzielski - we need to add `eslint-plugin-jsdoc` to a list of peer dependencies in the package.json.

**Comment by niedzielski** [comment]
@Pchelolo, thank you for reviewing! I apologize if I'm missing something obvious but `eslint-plugin-jsdoc` appears to be properly placed in both `devDependencies` and [`peerDependencies`](https://github.com/wikimedia/eslint-config-node-services/pull/9/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R27). Would you mind double checking this appears correctly?

**Comment by Pchelolo** [comment]
Thank you @niedzielski for the PR. Sorry it took me so long to get to this

**Pull request merged**